menu/index.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs-extra');
+var twigger = require('./lib/twig-compiler');
```

Comment:
This should point to pattern-importer's version instead

`var twigger = require('pattern-importer').twigCompiler();`

and you can now install pattern-importer from npm:

`npm install --save pattern-importer`